Routs/tasksRouts.js

```diff
@@ -0,0 +1,120 @@
+module.exports = function(app, db){
+    const {v4: uuidv4} = require('uuid')
+
+    function GenerateUUID()
```

Comment:
there is no additional value to wrap the uuidv4 in a function that just call it, call it yourself instead calling GenerateUUID